pkg/timestream/runningQuery.go

```diff
@@ -0,0 +1,52 @@
+package timestream
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/grafana/timestream-datasource/pkg/models"
+)
+
+type openQuery struct {
+	queryId string
+	query   *models.QueryModel
+	ds      *timestreamDS
+}
+
+func (q *openQuery) doStream(ctx context.Context, sender *backend.StreamSender) error {
+	backend.Logger.Info("Starting stream for", "queryId", q.queryId, "token", q.query.NextToken)
+	timer := time.NewTimer(time.Second * 2)
```

Comment:
I'm not sure what interval makes the most sense -- since the query will block, something short seems reasonable